src/CommunityToolkit.Maui/Extensions/ValueConverterExtension.shared.cs

```diff
@@ -75,9 +75,9 @@ static bool IsValidNullableValueType(Type targetType)
 	private protected static TValue ConvertValue<TValue>(object? value) => value switch
 	{
 		null when IsNullable<TValue>() => default,
-		null when !IsNullable<TValue>() => throw new ArgumentNullException(nameof(value), $"Value cannot be null because {nameof(TValue)} is not nullable"),
+		null when !IsNullable<TValue>() => throw new ArgumentNullException(nameof(value), $"CurrentRating cannot be null because {nameof(TValue)} is not nullable"),
```

Comment:
Looks like a Find + Replace typo
```suggestion
		null when !IsNullable<TValue>() => throw new ArgumentNullException(nameof(value), $"Value cannot be null because {nameof(TValue)} is not nullable"),
```